Assets/Scripts/HealthBar.cs

```diff
@@ -9,9 +9,11 @@ public class HealthBar : MonoBehaviour
     public Slider slider;
     public Gradient gradient;
     public Image fill;
+    public int health;
```

Comment:
Is this here to track player health? If so we already do it here:

https://github.com/AndreiBaetica/Dungeoneer/blob/master/Assets/Scripts/Controllers/CharController.cs#L12